test/tests/redirects.test.js

```diff
@@ -0,0 +1,38 @@
+import { expect } from 'chai'
+import request from 'request'
+import { bootServer, mockProxy, mockApi } from '../utils'
+
+describe('Handling redirects', () => {
+
+  let tapestry = null
+  let config = {
+    redirectPaths: {
+      '/redirect/from/this-path': '/about/home'
```

Comment:
Just a question about this API, would an array with `from` and `to` be cleaner? Or unnecessarily verbose?
```
redirectPaths: [{
  from: '/',
  to: '/'
}]
```